diff --git a/dist-cjs/lib/license/LicenseManager.js b/dist-cjs/lib/license/LicenseManager.js
index 0c139919dee38d439fd066d1ee61472f4336e282..d2beaaf1ca0618554b2bf5d813fc712b1bec03d1 100644
--- a/dist-cjs/lib/license/LicenseManager.js
+++ b/dist-cjs/lib/license/LicenseManager.js
@@ -82,7 +82,7 @@ class LicenseManager {
     });
   }
   getIsDevelopment() {
-    return !["https:", "vscode-webview:"].includes(window.location.protocol) || window.location.hostname === "localhost" || process.env.NODE_ENV !== "production";
+    return true;
   }
   getTrackType(result, licenseState) {
     if (licenseState === "unlicensed-production") {
@@ -324,7 +324,7 @@ class LicenseManager {
 function getLicenseState(result, outputMessages, isDevelopment) {
   if (!result.isLicenseParseable) {
     if (isDevelopment) {
-      return "unlicensed";
+      return "licensed";
     }
     if (result.reason === "no-key-provided") {
       outputMessages([
diff --git a/dist-esm/lib/license/LicenseManager.mjs b/dist-esm/lib/license/LicenseManager.mjs
index 2dc2d8989141aa72ec67c2290eaccb71832d662f..9d864296eb5f8a82b100cac8c420648856491f13 100644
--- a/dist-esm/lib/license/LicenseManager.mjs
+++ b/dist-esm/lib/license/LicenseManager.mjs
@@ -56,7 +56,7 @@ class LicenseManager {
     });
   }
   getIsDevelopment() {
-    return !["https:", "vscode-webview:"].includes(window.location.protocol) || window.location.hostname === "localhost" || process.env.NODE_ENV !== "production";
+    return true;
   }
   getTrackType(result, licenseState) {
     if (licenseState === "unlicensed-production") {
@@ -298,7 +298,7 @@ class LicenseManager {
 function getLicenseState(result, outputMessages, isDevelopment) {
   if (!result.isLicenseParseable) {
     if (isDevelopment) {
-      return "unlicensed";
+      return "licensed";
     }
     if (result.reason === "no-key-provided") {
       outputMessages([
